src/components/svgs/icons/badge-check-fill-icon/index.tsx

```diff
@@ -1,10 +1,10 @@
-export const BadgeCheckFillIcon = () => {
-  return (
-    <svg className="w-full" viewBox="0 0 14 15" fill="none" xmlns="http://www.w3.org/2000/svg">
-      <path
-        d="M3.73363 1.77309C4.29632 1.72818 4.8305 1.50692 5.26013 1.1408C6.26271 0.286402 7.73727 0.286402 8.73985 1.1408C9.16947 1.50692 9.70366 1.72818 10.2664 1.77309C11.5794 1.87787 12.6221 2.92056 12.7269 4.23363C12.7718 4.79632 12.993 5.3305 13.3592 5.76013C14.2136 6.76271 14.2136 8.23727 13.3592 9.23985C12.993 9.66947 12.7718 10.2037 12.7269 10.7664C12.6221 12.0794 11.5794 13.1221 10.2664 13.2269C9.70366 13.2718 9.16947 13.493 8.73985 13.8592C7.73727 14.7136 6.26271 14.7136 5.26013 13.8592C4.8305 13.493 4.29632 13.2718 3.73363 13.2269C2.42056 13.1221 1.37787 12.0794 1.27309 10.7664C1.22818 10.2037 1.00692 9.66947 0.640795 9.23985C-0.213598 8.23727 -0.213598 6.76271 0.640795 5.76013C1.00692 5.3305 1.22818 4.79632 1.27309 4.23363C1.37787 2.92056 2.42056 1.87787 3.73363 1.77309ZM10.2437 6.36872C10.5854 6.02701 10.5854 5.47299 10.2437 5.13128C9.90202 4.78957 9.34797 4.78957 9.00629 5.13128L6.125 8.01257L4.99372 6.88128C4.65201 6.53957 4.09799 6.53957 3.75628 6.88128C3.41457 7.22299 3.41457 7.77702 3.75628 8.11871L5.50628 9.86871C5.84799 10.2104 6.40201 10.2104 6.74372 9.86871L10.2437 6.36872Z"
-        fill="currentColor"
-      />
-    </svg>
-  )
-}
+export const BadgeCheckFillIcon = () => {
```

Comment:
I would recommend you to use functions instead of arrow-functions.

```suggestion
export function BadgeCheckFillIcon() {
```

Reply:
thank you, but in which cases should I do it? 🤔